### PR TITLE
Feature/582/Show cut-off controls

### LINKED ISSUE #582: If the window is too small horizontally the controls get hidden
# Bug

Controls in our file and metric bar disappear when the window is too small horizontally. The user now has no way to know that content is missing.

## Expected Behavior

The user knows that content is missing or there is no way that content is missing. Please do not show a burger menu.

## Actual Behavior

Content is missing.

### PR DESCRIPTION
**Pull request by alexhunziker**
# Show cut-off controls if window is to small

closes #582

## Description

- Shows cut off controls in filePanel/toolBar if window size is too small
(same issue for ribbonBar is closed in #526)

## Definition of Done

A task/pull request will not be considered to be complete until all these items can be checked off.

- [x] **All** requirements mentioned in the issue are implemented
- [x] Does match the Code of Conduct and the Contribution file
- [x] Task has its own **GitHub issue** (something it is solving)
  - [x] Issue number is **included in the commit messages** for traceability
- [x] **Update the README.md** with any changes/additions made
- [x] **Update the CHANGELOG.md** with any changes/additions made
- [x] **Enough test coverage to ensure that uncovered changes do not break functionality**
- [x] **All tests pass**
- [x] **Descriptive pull request text**, answering:
  - What problem/issue are you fixing?
  - What does this PR implement and how?
- [x] **Assign your PR to someone** for a code review
  - This person _will be contacted **first**_ if a bug is introduced into `master`
- [x] **Manual testing** did not fail

### CODE REVIEW

File: CHANGELOG.md [hunk 1]
```diff
@@ -11,6 +11,8 @@ and this project adheres to [Semantic Versioning](http://semver.org/)
 
 ### Changed
 
+- toolBar now shows now cut-off controls if window is too small #582
```
Comment (NearW) [inline CHANGELOG.md]: https://media3.giphy.com/media/CiYImHHBivpAs/giphy.gif

**Comment by alexhunziker** [comment]
@NearW wait i am confused; how does it look on your machine? On my browser it looks like this:
grafik
Isn't that pretty much the behaviour you describe?

Regarding the settings button, so now it moves down if there is no space. Personally i am not a fan of overlapping controls, it looks glitchy... generally our current approach potentially prevents the user from using some functionality on both panels, so personally I would even be fine with hiding the settings button.

File: CHANGELOG.md (hunk 1 above)
Comment (alexhunziker) [reply CHANGELOG.md]: well the toolBar now shows now... ok i updated that ;)

**Comment by alschmut** [comment]
I would either not show the settings button or overlap the filePanel

**Comment by alexhunziker** [comment]
ok, i changed the settings, now it is floating over the other components if there is not enough space. While still not a fan of overlapping controls, i realized that we have exactly this behaviour with the expand-button, so therefore it is only consistent to do that with the settings-button

**Comment by NearW** [comment]
scale_filebar

I was talking about scaling it REALLY small. The question here is: Should we support that kind of resolution? lol

**Comment by alschmut** [comment]
@NearW but your gif shows an outdated version ;)

**Pull request merged**